protostar/testing/test_case_runners/estimate_fee.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional
+
+from protostar.testing.starkware.execution_resources_summary import (
+    ExecutionResourcesSummary,
+)
+
+
+def estimate_fee(
```

Comment:
Please use existing functions for that, discussed on Slack

Reply:
[use existing function](https://github.com/software-mansion/protostar/pull/1013/commits/8b719e9450f2ceffc27fc9b0ec651be237597938)